src/sparseml/keras/optim/manager.py

```diff
@@ -22,6 +22,12 @@
 
 import tensorflow as tf
 
+
+try:
```

Comment:
We shouldn't be repeating this everywhere. Please move this to a utils function and import keras from that utility like we did for tf_compat